src/openrct2-ui/windows/Guest.cpp

```diff
@@ -1996,7 +2002,9 @@ static rct_string_id window_guest_inventory_format_item(Peep* peep, int32_t item
                         ft.Rewind();
                         ft.Increment(6);
                         ft.Add<rct_string_id>(STR_PEEP_INVENTORY_VOUCHER_RIDE_FREE);
-                        ride->FormatNameTo(gCommonFormatArgs + 8);
+                        ft = Formatter::Common();
```

Comment:
You already have ft at the correct location no need for the rewind and increment